packages/fsapp/src/beta-app/utils.base.tsx

```diff
@@ -26,3 +28,7 @@ export const promisedEntries = async (data: Record<string, unknown>) => {
     )
   );
 };
+
+export type Mutable<T> = { -readonly [P in keyof T]?: T[P] };
+
+export const NOTHING_BURGER: FC = ({ children }) => <>{children}</>;
```

Comment:
We might want to call this something like FC_NOOP_WRAPPER or other. 😛

Reply:
😅 Naming things is hard.